l10n_it_mis_reports_pl_bs/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,10 @@
+To configure this module, you need to go to
+Accounting > Reporting > MIS Reports and create report instance
```

Comment:
```suggestion
*Accounting > Reporting > MIS Reporting > MIS Reports* and create report instance
```

Corsivo aggiunto giusto per aumentare la leggibilità.

P.s.: eventualmente per le frecce a dx si potrebbe utilizzare il carattere → (AltGr + i) e usarlo come standard per i percorsi nella documentazione (giusto un'idea in previsione futura).